_maps/yogstation/RandomRuins/StationRuins/BoxStation/bar_disco.dmm

```diff
@@ -0,0 +1,1401 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/turf/template_noop,
+/area/template_noop)
+"ab" = (
+/turf/closed/wall{
+	canSmoothWith = list(/turf/closed/wall,/obj/structure/falsewall,/obj/structure/falsewall/reinforced,/turf/closed/wall/rust,/turf/closed/wall/clockwork);
+	icon = 'icons/turf/walls/abductor_wall.dmi';
+	icon_state = "abductor"
+	},
+/area/crew_quarters/bar)
+"ac" = (
+/obj/machinery/door/airlock{
+	name = "Theatre Backstage";
+	req_access_txt = "46"
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer1,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer3,
+/turf/open/floor/plasteel,
+/area/crew_quarters/theatre)
+"ad" = (
+/obj/item/storage/secure/safe,
+/turf/closed/wall{
+	canSmoothWith = list(/turf/closed/wall,/obj/structure/falsewall,/obj/structure/falsewall/reinforced,/turf/closed/wall/rust,/turf/closed/wall/clockwork);
+	icon = 'icons/turf/walls/abductor_wall.dmi';
+	icon_state = "abductor"
+	},
+/area/crew_quarters/bar)
+"ae" = (
+/obj/machinery/door/airlock/maintenance{
+	name = "Bar Maintenance";
+	req_access_txt = "12"
+	},
+/obj/structure/disposalpipe/segment,
+/turf/open/floor/plating,
+/area/crew_quarters/bar)
+"af" = (
+/obj/item/reagent_containers/food/drinks/shaker,
+/obj/structure/table/wood,
+/obj/item/stack/sheet/metal{
+	amount = 50
+	},
+/obj/item/stack/sheet/glass{
+	amount = 50
+	},
+/obj/item/stack/cable_coil,
+/obj/item/gun/ballistic/revolver/doublebarrel,
+/turf/open/floor/plasteel/ameridiner,
+/area/crew_quarters/bar)
+"ag" = (
+/obj/structure/disposalpipe/segment,
+/turf/open/floor/plasteel/ameridiner,
+/area/crew_quarters/bar)
+"ah" = (
+/obj/structure/sink/kitchen{
+	pixel_y = 28
+	},
+/turf/open/floor/plasteel/ameridiner,
+/area/crew_quarters/bar)
+"ai" = (
+/turf/closed/wall,
+/area/crew_quarters/bar)
+"aj" = (
+/obj/machinery/door/window/southleft{
+	base_state = "left";
+	dir = 2;
+	icon_state = "left";
+	name = "Bar Delivery";
+	req_access_txt = "25"
+	},
+/obj/effect/turf_decal/delivery,
+/turf/open/floor/plasteel,
+/area/crew_quarters/bar)
+"ak" = (
+/turf/closed/wall,
+/area/crew_quarters/kitchen)
+"al" = (
+/obj/machinery/door/airlock/maintenance{
+	name = "Kitchen Maintenance";
+	req_access_txt = "28"
+	},
+/obj/structure/disposalpipe/segment,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer1,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer3,
+/turf/open/floor/plating,
+/area/crew_quarters/kitchen)
+"am" = (
+/obj/machinery/navbeacon{
+	codes_txt = "delivery;dir=2";
+	freq = 1400;
+	location = "Kitchen"
+	},
+/obj/structure/plasticflaps{
+	opacity = 1
+	},
+/turf/open/floor/plasteel,
+/area/crew_quarters/kitchen)
+"an" = (
+/turf/open/floor/mineral/plasma,
```

Comment:
yeah I made the bar have plasma windows for that exact reason and I did it cause its supposed to be kinda like a bar version of disco inferno death by disco am I right?

Reply:
there's also a shuttle that's an entirely airless hull of plating, but if I made a bar like that because it'd be like the shuttle, I'd get told that's a shit reason :)

Reply:
eh true